plugins/com.google.cloud.tools.eclipse.preferences/src/com/google/cloud/tools/eclipse/preferences/messages.properties

```diff
@@ -0,0 +1,4 @@
+FIELD_EDITOR_ANALYTICS_DISCLAIMER=Your use of this plug-in is subject to the Apache 2.0 software license and the <a href="privacy-policy">Google Privacy Policy</a>.
```

Comment:
I don't think we need to mention the license here. We should take it out so the privacy policy is more prominent. What we want is something like "Share anonymous usage statistics of Google Cloud Tools with Google subject to the Google Privacy Policy."